apollo-ios-codegen/Sources/GraphQLCompiler/CompilationResult.swift

```diff
@@ -255,6 +279,18 @@ public final class CompilationResult: JavaScriptObjectDecodable {
     public static func ==(lhs: FragmentDefinition, rhs: FragmentDefinition) -> Bool {
       return lhs.name == rhs.name
     }
+    
+    private static func getImportModuleNames(directives: [Directive]?, 
```

Comment:
I can add this as a global function in order to avoid any code duplication. For now i left this helper inside the corresponding class.